File: assets/integrations/shipping-services-api-sample-code/ShippingClientCSharp/ShippingClientCSharp/frmMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using ShippingClientCSharp.PrintLabel;
using ShippingClientCSharp.CreateShipments;
using ShippingClientCSharp.CreatePickup;
using ShippingClientCSharp.CancelPickup;

namespace ShippingClientCSharp
{
    public partial class frmMain : Form
    {
        public frmMain()
        {
            InitializeComponent();
        }

        private void btnCreateShipments_Click(object sender, EventArgs e)
        {
            using (Form1 _frmShipments = new Form1())
            {
                _frmShipments.ShowDialog(this);
            }
        }

        private void btnPrintLabel_Click(object sender, EventArgs e)
        {
            using (frmPrintLabel _frmLabel = new frmPrintLabel())
            {
                _frmLabel.ShowDialog(this);
            }
        }

        private void btnCreatePickup_Click(object sender, EventArgs e)
        {
            using (frmPickup _frmPickup = new frmPickup())
            {
                _frmPickup.ShowDialog(this);
            }
        }

        private void btnCancelPickup_Click(object sender, EventArgs e)
        {
            using (frmCancelPickup _frmCancelPickup = new frmCancelPickup())
            {
                _frmCancelPickup.ShowDialog(this);
            }
        }

        private void btnExit_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}

File: assets/integrations/shipping-services-api-sample-code/ShippingClient/ShippingClient/ShippingClient/frmMain.vb
Imports ShippingClient.ShippingReference

Public Class frmMain

    Private Sub btnCreateShipments_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCreateShipments.Click
        Using _frmShipments As New Form1
            _frmShipments.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnPrintLabel_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnPrintLabel.Click
        Using _frmLabel As New frmPrintLabel
            _frmLabel.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnExit_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnExit.Click
        Me.Close()
    End Sub

    Private Sub btnCreatePickup_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCreatePickup.Click
        Using _frmPickup As New frmPickup
            _frmPickup.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnCancelPickup_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCancelPickup.Click
        Using _frmCancelPickup As New frmCancelPickup
            _frmCancelPickup.ShowDialog(Me)
        End Using
    End Sub
End Class